src/features/recipients/components/GetPublicKey.tsx

```diff
@@ -0,0 +1,26 @@
+import { Button, Text } from '@chakra-ui/react';
+
+interface GetPublicKeyProps {
+  onClickGetPublicKey?: () => void;
+  isLoading?: boolean;
+}
+
+export function GetPublicKey({ onClickGetPublicKey, isLoading }: GetPublicKeyProps) {
+  return (
+    <>
+      <Text variant="secondary">
+        Click the button below to digitally sign a message with your connected wallet so that
```

Comment:
How does `Click the button below to securely sign a message with your connected wallet` read? 
ie "securely" vs "digitally"